nucleus/payara-modules/opentracing-adapter/src/main/java/fish/payara/opentracing/OpenTracingService.java

```diff
@@ -76,10 +78,17 @@
     // The tracer instances
     private static final Map<String, Tracer> tracers = new ConcurrentHashMap<>();
     
+    // The name of the Corba RMI Tracer
+    public static final String PAYARA_CORBA_RMI_TRACER_NAME = "__PAYARA_CORBA_RMI";
+
     @PostConstruct
     void postConstruct() {
         // Listen for events
-        Globals.getDefaultBaseServiceLocator().getService(Events.class).register(this);
+        Events events = getFromServiceHandle(Globals.getDefaultBaseServiceLocator(), Events.class);
```

Comment:
Is there a reason to not have these injected and kept in a field?

Reply:
Injection attempts to start the service, something not necessarily desired when running on a Java SE client.